x/auth/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=cosmos-sdk-x-auth
```

Comment:
https://sonarcloud.io/project/configuration?id=cosmos-sdk-x-auth

Reply:
So you mean that given that Sonar Cloud hasn't yet accepted this module (aka before merging this PR), that it'll fail? If so, no biggie and thanks, otherwise how do we get it recognized?